<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shutterControl</class>
 <widget class="QDialog" name="shutterControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>shutterControl</string>
  </property>
  <widget class="QPushButton" name="openShutter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>快门打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeShutter">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>121</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>121</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string>快门关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="conectShutter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>连接快门</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
